src/server/lib/kafka-consumer.js

```diff
@@ -16,25 +16,34 @@ let kafka = require('kafka-node'),
  * @returns {string}
  */
 function createGroupId(topic) {
-	return 'nodejs-' + topic + '-' + Math.random();
+	return `nodejs-${topic}-${Math.random()}`;
 }
 
 /**
- * Gets a new Kafka client.  We need to do this for each new connection, according to Kafka best practices.
+ * Gets a new Kafka ConsumerGroup.  We need to do this for each new connection, according to Kafka best practices.
  *
- * @returns A promise to return a KafkaClient.
+ * @returns A promise to return a ConsumerGroup.
  */
-function getClient() {
-	var client = new kafka.Client(config.kafka.zookeeper);
-	return q.resolve(client);
+function getConsumer(topic, groupId) {
+	let consumer = new kafka.ConsumerGroup(
+		{
+			host: config.kafka.zookeeper,
+			groupId: groupId,
+			fromOffset: 'latest',
```

Comment:
so I changed this to expose all these options as a parameter of the KafkaConsumer constructor, that will override these defaults if you specify a value for these properties.